client/src/components/Typography/TextLabel.js

```diff
@@ -27,8 +27,12 @@ const TextLabel = styled(Text)`
     color: ${(props) => props.color || "black"};
     font-size: ${(props) => props.size || theme.typography.size.medium};
     font-weight: ${(props) => props.weight || "normal"};
-    display: ${(props) => (!props.block ? "" : "block")};
     text-align: ${(props) => props.textAlign || "left"};
+    display: none;
+
+    @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+      display: block;
+    }
   }
```

Comment:
Hey there :) these display rules led another undesired result in ProfileComponents.js which effects Profile.js
Please check 

```
export const LocationMobileDiv = styled(TextLabel)`
  align-self: center;
  color: ${colors.darkGray};
  @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
    display: none;
  }
`;

export const NameDiv = styled(TextLabel)`
  align-self: center;
  @media screen and (min-width: ${mq.tablet.narrow.minWidth}) {
    display: flex;
    width: 100%;
    align-self: flex-start;
    &.ant-typography {
      font-size: 3rem;
      font-weight: bold;
    }
  }
`;
```
This is from rc/v1.1

![1](https://user-images.githubusercontent.com/34690158/88493166-5da13700-cfb8-11ea-8d44-e03e10d727f9.gif)

And this is from https://staging.fightpandemics.work/

![2020-07-27_03-23-34](https://user-images.githubusercontent.com/34690158/88493235-b1138500-cfb8-11ea-8de6-6c972a32fb78.gif)